src/app/view/components/_common/dot-page-selector/dot-page-selector.component.html

```diff
@@ -0,0 +1,28 @@
+<span dotMdInputtext [label]="label" *ngIf="floatingLabel; else placeholded">
+    <p-autoComplete
+        (completeMethod)="search($event.query)"
+        (onClear)="onClear($event)"
+        (onSelect)="onSelect($event)"
+        [inputStyle]="{'width': '100%', 'padding': '0'}"
+        [minLength]="3"
+        [ngModel]="val"
+        [style]="style"
+        [suggestions]="results"
+        field="url">
+    </p-autoComplete>
+</span>
+
+<ng-template #placeholded>
+    <p-autoComplete
+        (completeMethod)="search($event.query)"
+        (onClear)="onClear($event)"
+        (onSelect)="onSelect($event)"
+        [inputStyle]="{'width': '100%', 'padding': '0'}"
+        [minLength]="3"
+        [ngModel]="val"
+        [placeholder]="label"
+        [style]="style"
+        [suggestions]="results"
+        field="url">
+    </p-autoComplete>
+</ng-template>
```

Comment:
All the <p-autoComplete is repeat here, I remove the floatingLabel attribute and if we need it in the future then we'll add it

Reply:
100% disagreed, I don't want a field that is attached to a floating label effect, the floating label effect is optional, I want my fields to be, act an look as native as possible.